defaults/k8s_local_kind_config.yaml

```diff
@@ -36,3 +36,5 @@ backtrace_decoding: false
 append_scylla_args: '--abort-on-lsa-bad-alloc 1 --abort-on-internal-error 1 --abort-on-ebadf 1 --enable-sstable-key-validation 1'
 docker_image: ''
 backup_bucket_location: 'minio-bucket'
+
+k8s_enable_tls: true
```

Comment:
so we'll now enabled it by default ? (regardless of what we talked that we'll still want to test it without it for 1.8.0 release ?)

Reply:
or you enabled it here only for testing ?